components/camel-test/src/main/java/org/apache/camel/test/junit4/LanguageTestSupport.java

```diff
@@ -85,8 +85,9 @@ protected void assertExpression(String expressionText, String expectedValue, Str
         } else {
             value = expression.evaluate(exchange, Object.class);
         }
-
-        log.debug("Evaluated expression: " + expression + " on exchange: " + exchange + " result: " + value);
+        if (log.isDebugEnabled()) {
```

Comment:
The rest is just unit tests and tooling so its not as important as runtime camel-core etc. However the changes are fine to do, but maybe favor using `{}` style

Reply:
In general, you do not recommend increasing complexity with the if statement. right?